recipes/essreduce/recipe.yaml

```
@@ -0,0 +1,56 @@
+context:
+  name: essreduce
+  version: 25.7.1
+  python_min: "3.10"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/essreduce-${{ version }}.tar.gz
+  sha256: cdb389c585619a8b22516e56f8064ede5cf2fac24de22fec9d5af5469c065e47
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  python:
+    entry_points:
+      - grow-nexus = ess.reduce.scripts.grow_nexus:main
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - setuptools >=68
+    - setuptools-scm >=8.0
+    - pip
+  run:
+    - python >=${{ python_min }} 
+    - sciline >=25.05.1
+    - scipp >=25.01.0
+    - scippneutron >=25.02.0
+    - scippnexus >=24.11.0
+
+tests:
+  - python:
+      imports:
+        - ess
+      pip_check: true
+      python_version: ${{ python_min }}.*
+  - requirements:
+      run:
+        - pip
```

```suggestion
        - python ${{ python_min }}.*
```

You don't need `pip` here since the pip check is handled above, but you do need to specify the python version separately for the script.